### PR TITLE
Implement Delete and Search for Products

### PR DESCRIPTION
**Pull request by merecka**
Added a delete button to remove Products.  
Added a Search bar in the Manage Products page to search (filter) for Products.

### CODE REVIEW

File: src/app/product.service.ts [hunk 1]
```diff
@@ -33,4 +33,8 @@ export class ProductService {
   update(productId, product): void {
     this.database.object('/products/' + productId).update(product);
   }
+
+  delete(productId) {
```
Comment (BarretteBanner) [inline src/app/product.service.ts]: You can set type to void and remove the return

Comment (merecka) [reply src/app/product.service.ts]: Done

File: src/app/admin/product-form/product-form.component.ts [hunk 2]
```diff
@@ -38,4 +38,10 @@ export class ProductFormComponent {
 
     this.router.navigate(['/admin/products']);
   }
+
+  delete() {
+    if (!confirm('Are you sure you want to delete this product?')) return;
```
Comment (ahaly92) [inline src/app/admin/product-form/product-form.component.ts]: can you pull this error message into a const at the top of the file

**Review by ahaly92 — APPROVED** [review]
small comment then rebase and ship that shit

File: src/app/admin/product-form/product-form.component.ts (hunk 2 above)
Comment (merecka) [reply src/app/admin/product-form/product-form.component.ts]: Done.

**Pull request merged**